parsec/core/core_events.py

```diff
@@ -34,3 +34,16 @@ class CoreEvent(Enum):
     # Others
     SHARING_UPDATED = "sharing.updated"
     USERFS_UPDATED = "userfs.updated"
+
+
+class FSEntryUpdatedReason(IntEnum):
```

Comment:
Why `IntEnum` and not `Enum` ? Given we don't do serialization on this type, `Enum` seems good enough to me

Besides instead of using `auto` (which give an int as a value), we should use a string a value (to be consistent with wath is done for all the other enums in the codebase):
```
class FSEntryUpdatedReason(Enum):
    ENTRY_RENAME = "ENTRY_RENAME"
...
```